src/sentry/models/activity.py

```diff
@@ -58,6 +58,7 @@ def get_activities_for_group(self, group: Group, num: int) -> Sequence[Group]:
                 project=group.project,
                 group=group,
                 type=ActivityType.FIRST_SEEN.value,
+                data={"priority": group.priority},
```

Comment:
We actually want the _first_ priority assigned to the issue, so you'll want to get it from `group.get_event_metadata().get("initial_priority")`